html/semantics/scripting-1/the-script-element/module/dynamic-import/string-compilation-integrity-classic.sub.html

```diff
@@ -4,8 +4,8 @@
 <link rel="author" title="Domenic Denicola" href="mailto:d@domenic.me">
 <meta http-equiv="Content-Security-Policy" content="require-sri-for script">
 
-<script src="/resources/testharness.js" integrity="sha384-4Nybydhnr3tOpv1yrTkDxu3RFpnxWAxlU5kGn7c8ebKvh1iUdfVMjqP6jf0dacrV"></script>
-<script src="/resources/testharnessreport.js" integrity="sha384-GOnHxuyo+nnsFAe4enY+RAl4/+w5NPMJPCQiDroTjxtR7ndRz7Uan8vNbM2qWKmU"></script>
+<script src="/resources/testharness.js" integrity="sha384-{{sha384[resources/testharness.js]}}"></script>
```

Comment:
Rather than using testharness.js, could we not add another file that doesn't ever change?

Reply:
@domenic, WDYT?

Reply:
I think you misunderstand the test. If you have a CSP stating `require-sri-for script` all scripts in the document need to have integrity defined. That includes the harness scripts.